db/seeds.rb

```diff
@@ -40,6 +36,15 @@ def describe_registration_period(ay)
   print_seed_info("#{ay.year} (running from #{ay.started_on} until #{ay.finished_on})", indent: 2)
 end
 
+def describe_lead_provider(lp, years)
+  years_description = if years.any?
+                        Colourize.text(years.map(&:year).join(', '), :green)
+                      else
+                        Colourize.text('inactive', :red)
+                      end
+  print_seed_info("#{lp.name} (#{years_description})", indent: 2)
+end
```

Comment:
This prints:

![image](https://github.com/user-attachments/assets/d94c3fca-303d-4061-a75c-3ce7aaf72dc7)